provision/minikube/Taskfile.yaml

```diff
@@ -13,7 +13,15 @@ vars:
   KC_DB_POOL_MIN_SIZE: '{{default "5" .KC_DB_POOL_MIN_SIZE}}'
   KC_STORAGE: '{{default "" .KC_STORAGE}}'
   KC_DATABASE: '{{default "postgres" .KC_DATABASE}}'
+  KC_OPERATOR_TAG: '{{default "nightly" .KC_OPERATOR_TAG}}'
   KC_CONTAINER_IMAGE: '{{default "" .KC_CONTAINER_IMAGE}}'
+  KC_SCALE: '{{default "1" .KB_SCALE}}'
```

Comment:
looks like this should be `KC_SCALE: '{{default "1" .KC_SCALE}}'` seems like a typo for KB_SCALE